wordpress-mobile/WordPress-iOS
# Comment Detail Redesign: Implement trash & delete

Fixes #18999 

This is the final PR to be merged on the feature branch. After this one, we should test the feature branch all together and update the Release Notes.

To test:
1. Log in to a WP Account
2. In My Site, select Menu
3. Comments
4. Select any comment
5. Tap on trash/delete button.
6. Validate if trash moves the comment.
7. Tap on one of the status options (Approve/Pending/Spam) to see if button updates from Delete to Trash again.
8. Verify if Delete permanently deletes the comment.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
